src/cmd/cmdgazebo.rb.in

```diff
@@ -40,76 +40,85 @@ COMMON_OPTIONS =
+  "  -r                           Run simulation on start.                         \n"\
   "\n"\
-  "  -s                          Run only the server (headless mode). This  \n"\
-  "                              overrides -g, if it is also present.       \n"\
+  "  -s                           Run only the server (headless mode). This        \n"\
+  "                               overrides -g, if it is also present.             \n"\
   "\n"\
-  "  -v [ --verbose ] [arg]      Adjust the level of console output (0~4).  \n"\
-  "                              The default verbosity is 1, use -v without \n"\
-  "                              arguments for level 3.                     \n"\
+  "  -v [ --verbose ] [arg]       Adjust the level of console output (0~4).        \n"\
+  "                               The default verbosity is 1, use -v without       \n"\
+  "                               arguments for level 3.                           \n"\
   "\n"\
-  "  --gui-config [arg]          Ignition GUI configuration file to load.   \n"\
-  "                              If no config is given, the configuration in\n"\
-  "                              the SDF file is used. And if that's not    \n"\
-  "                              provided, the default installed config is  \n"\
-  "                              used.                                      \n"\
+  "  --gui-config [arg]           Ignition GUI configuration file to load.         \n"\
+  "                               If no config is given, the configuration in      \n"\
+  "                               the SDF file is used. And if that's not          \n"\
+  "                               provided, the default installed config is        \n"\
+  "                               used.                                            \n"\
   "\n"\
-  "  --physics-engine [arg]      Ignition Physics engine plugin to load.    \n"\
-  "                              Gazebo will use DART by default.           \n"\
-  "                              (ignition-physics-dartsim-plugin)          \n"\
-  "                              Make sure custom plugins are in            \n"\
-  "                              IGN_GAZEBO_PHYSICS_ENGINE_PATH.            \n"\
+  "  --physics-engine [arg]       Ignition Physics engine plugin to load.          \n"\
+  "                               Gazebo will use DART by default.                 \n"\
+  "                               (ignition-physics-dartsim-plugin)                \n"\
+  "                               Make sure custom plugins are in                  \n"\
+  "                               IGN_GAZEBO_PHYSICS_ENGINE_PATH.                  \n"\
   "\n"\
-  "  --version                   Print Gazebo version information.          \n"\
+  "  --render-engine [arg]        Ignition Rendering engine plugin to load.        \n"\
+  "                               Gazebo will use OGRE2 by default.                \n"\
```

Comment:
provide the exact string value that's used? `Gazebo will use OGRE2 by default. (ogre2)`. Mainly just to be consistent with the `--physics-engine` arg description 

Just to confirm, I remember you mentioned we could support specifying the render engine using both the engine name and plugin name (which we could do later)?  I like specifying by the engine name which is less error-prone but we should document the inconsistency if we don't plan to address it in the near future.

Reply:
I agree that the engine name is more user friendly, but the plugin name is more general for when we support custom engines. So I think the end goal would be:

* Consistency across physics and rendering
* Support for custom engine plugins, referring to them by the shared library name:
    * `--render-engine MyCustomEngine`
    * `--physics-engine MyCustomEngine`
* Support for engine plugins that are shipped with the core libraries either by their shared library name
    * `--render-engine ignition-rendering-ogre2-plugin`
    * `--render-engine ignition-physics-dartsim-plugin`
    or their short name
    * `--render-engine ogre`
    * `--render-engine tpesim`
  To achieve this, we can always try to wrap the passed string into `ignition-[physics/rendering]-[name]-plugin`.

Reply:
I agree with all the points above, I was wondering if you have an idea as to how to support binary names?  Currently, specifying the regular name of the rendering engine is supported given the call here https://github.com/ignitionrobotics/ign-gazebo/pull/331/files#diff-a15bf0b158d65eb38bd94081b23c4291R2218 which takes "ogre" or "ogre2" and translates that to the binary name https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo3/src/rendering/RenderUtil.cc#L1020 there, we would likely need to modify RenderUtil to now accept binary names, right?

Reply:
> I was wondering if you have an idea as to how to support binary names?

I'll add more info on this issue: https://github.com/ignitionrobotics/ign-rendering/issues/100

---

To be clear, I'd be ok proceeding with this PR as is, and adding support for loading the shared libraries later. I'd just make the change that @iche033 suggested above to say "ogre2" instead of "OGRE2".

Reply:
Sounds good, fixes made here 88cd727c47443ca8fa46d52294199c4136ff7be4